Program.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using Microsoft.VisualBasic.FileIO;
+using System.Collections.Generic;
+
+namespace SupportBank
+{
+    class Program
+    {
+        static Dictionary<string, Account> ReadFile() {
+
+            var accounts = new Dictionary<string, Account>();
+
+            var parser = new TextFieldParser(@"Transactions2014.csv");
+            parser.TextFieldType = FieldType.Delimited;
+            parser.SetDelimiters(",");
+
+            if (!parser.EndOfData)
```

Comment:
I prefer to use brackets for `if` statements, especially if they're split over several lines. That makes it more clear what's part of the `if` block 🙂 

And is it not possible to combine this with the `while` statement? If it's to skip the first line in the file then I would recommend adding a comment explaining that 🙂